### DISCUSSION TITLE
Yellow Warning Sign

### DISCUSSION THREAD

**Discussion by CameronBurley** [ohmyzsh/ohmyzsh#12348] [Help]
What is this yellow warning sign at the end of my line upon startup?
![IMG_6466](https://github.com/ohmyzsh/ohmyzsh/assets/64497025/61579025-b4ab-480a-8085-d12c2bc8fa2b)

**Comment by mcornella** [comment]
No way to know without more information. In which application did this show up, what is the context,...?

**Comment by robbyrussell** [comment]
@CameronBurley do you still see this? Going to close this for now
